resources/js/components/ExpRightNav.vue

```diff
@@ -34,20 +37,21 @@
 		   	<span class="fa fa-align-justify bg-white rightnavexpander"></span>			
 		   <h1>Help</h1>				                
 		</div>
-		<div class="containerR" style="display: none;" id="unkl">	
-		<!--    	 <input type="text" name="search" class="input-search input-dark" ><span class="fa fa-search serachicon "></span> -->           
+		<!-- <div class="containerR" style="display: none;" id="unkl">			
 		   	<span class="fa fa-align-justify bg-white rightnavexpander"></span>
-            <div v-if="toolState==true" v-for="tool in toolSizes"  :key="tool" @click="addactivate;changeApparatus(tool)"  v-bind:style="{width:tool+'px'}" class="box">
+            <div v-if="toolstate==true" v-for="tool in toolsizes"  :key="tool" @click="addactivate;changeApparatus(tool)"  v-bind:style="{width:tool+'px'}" class="box">
             		Size
         	</div> 			                
-		</div>
+		</div> -->
 	</div>
 </template>
 <script>
 	import vicelabtools from './vicelabExpEquip';
+	import electricity from './electricityEquipment';
```

Comment:
excellent, this is exactly what we mean. So, what's stopping you from renaming the previous code block like this ?